src/robot/utils/asserts.py

```diff
@@ -177,7 +177,7 @@ def assert_raises_with_msg(exc_class, expected_msg, callable_obj, *args,
 
 def assert_equal(first, second, msg=None, values=True, formatter=safe_str):
     """Fail if given objects are unequal as determined by the '==' operator."""
-    if not first == second:
+    if first != second:
```

Comment:
I was browsing through the changes and noticed this one. This change is _not_ ok. `==` and `!=` call different methods, `__eq__` and `__ne__`, respectively, and thus there's a significant difference between `... != ...` and `not ... == ...`. It typically doesn't matter, but it matters in this particular context.

This is a rather minor thing, but shows that formatting changes must be reviewed _very_ carefully to avoid accidental functional changes. We don't have time for that before RF 7.0 and, because we can miss something, I believe we'd be too late in RF 7.0 development cycle also otherwise.